.github/workflows/master.yml

```diff
@@ -184,6 +184,21 @@ jobs:
         export PATH="$CONDA_DIR/bin:$PATH"
         source activate test-environment
         ./dev/run-large-python-tests.sh
+    - name: Run database initialization tests
```

Comment:
Is it possible to run these on PRs that modify alembic or sqlalchemy files as well?

Reply:
Yes, it's possible. We can control when this check should be executed. The current implementation is "any PRs must pass this check", but we can change it to "PRs that affect alembic or sqlalchemy files must pass this check"

Reply:
Ah, I didn't realize this ran on every PR since I didn't see it in the set of tests run on this PR. Am I missing something?

Reply:
It's run on every PR now because it's executed in the python-large check.

Reply:
Ah, got it - missed the header. Thanks Haru!